src/game.cpp

```diff
@@ -997,9 +1009,17 @@ bool game::start_game()
                     success = true;
                     break;
                 }
-                if( success ) {
-                    v.v->name = "Bird Wreckage";
-                    break;
+            }
+            for( auto rider : v.v->get_riders() ) {
```

Comment:
This is outside the `if( success )` in order to damage riders of every vehicle, not just the one the player is "driving".